app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,7 @@
 class WelcomeController < ApplicationController
   def index
-    redirect_to event_path(Event.first)
+    if Event.first
+      redirect_to event_path(Event.first)
```

Comment:
I wouldn't really care about this, this is something which should be seeded and thus should not error in the app normally. If it does, it's a clear signal to take action.